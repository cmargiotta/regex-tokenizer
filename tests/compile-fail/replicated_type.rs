use regex_tokenizer::tokenizer;

fn main() {
    tokenizer! {
        Test

        "a" => A
        "b" => A
        "c" => C
    }
}
