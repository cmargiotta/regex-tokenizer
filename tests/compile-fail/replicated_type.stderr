error: Token types must be unique
 --> tests/compile-fail/replicated_type.rs:8:16
  |
8 |         "b" => A
  |                ^
